packages/server/README.md

```diff
@@ -17,6 +17,16 @@ docker run --env-file .env -p 22300:22300 joplin/server:latest
 
 This will start the server, which will listen on port **22300** on **localhost**. By default it will use SQLite, which allows you to test the app without setting up a database. To run it for production though, you'll want to connect the container to a database, as described below.
 
+## Supported docker tags
+
+The following tags are available:
+
+- `latest` are always the most recent released version
+- `beta` are always the most recent beta released version
```

Comment:
the singular should be used in the above to sentences.

`... is always the most ....`

Reply:
done